src/libraries/CashWrapper.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/trading/ICash.sol';
+import 'ROOT/Controlled.sol';
+import 'ROOT/Augur.sol';
+
+
+/**
+ * @title Provides modifiers which take care of Cash/ETH conversion
+ */
+contract CashWrapper is Controlled {
+    /**
+     * @dev Convert any ETH provided in the transaction into Cash before the function executes
+     */
+    modifier convertToCash() {
+        ethToCash();
+        _;
+        cashToETH();
+    }
```

Comment:
I wonder if we could get away with just a single modifier that works like this one and delete `convertFromCash`?  Is there any situation where converting from ETH at the start would be a problem but we need to convert to ETH at the end?

Reply:
I'd like that better. The Cash -> ETH modifier felt a little weird to me anyway.